docs/data/joy/getting-started/usage/usage.md

```diff
@@ -1,42 +1,109 @@
 # Usage
 
-<p class="description">Learn the basics of using Joy UI components in two quick steps.</p>
+<p class="description">Learn the basics of using Joy UI components.</p>
 
 ## Getting started
 
-### Set up the `CssVarsProvider` component
-
-Go to your `App.js` file and replace it with the code snippet below.
-
-You should see the text `Hello from Joy` being rendered on your browser.
+The following code snippet demonstrates a simple app that uses the Joy UI [Button](/joy-ui/react-button/) component:
 
 ```jsx
+import * as React from 'react';
 import { CssVarsProvider } from '@mui/joy/styles';
+import Button from '@mui/joy/Button';
 
-function App() {
-  return <CssVarsProvider>Hello from Joy</CssVarsProvider>;
+export default function App() {
+  return (
+    <CssVarsProvider>
+      <Button>Hello World</Button>
+    </CssVarsProvider>
+  );
 }
+```
+
+### CssVarsProvider
+
+In the example above, you can see that the Button component is nested within `<CssVarsProvider />`.
+This is not required, but it unlocks a whole host of customization options powered by CSS variables.
```

Comment:
@samuelsycamore All Joy UI components require `CssVarsProvider` at the root to provide CSS theme variables.

Reply:
Ah my mistake, I'm not sure where I got the idea that it was optional.